test/src/test/java/org/corfudb/runtime/clients/NettyCommTest.java

```diff
@@ -394,11 +394,13 @@ private void reloadedTrustManagerTestHelper(boolean replaceClientTrust) throws E
 
         if (replaceClientTrust) {
             Files.copy(clientTrustWithServer.toPath(), clientTrustFile.toPath(), StandardCopyOption.REPLACE_EXISTING);
+            //force reload to kick in
+            clientTrustFile.setLastModified(0);
```

Comment:
This entire second commit won't be needed if you unconditionally reload the trust store.